slsim/Deflectors/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .all_lens_galaxies import AllLensGalaxies as AllLensGalaxies
```

Comment:
just asking, is there a particular reason to put these import statements in the init file? This may be harder to maintain in the development to make sure the relevant classes and definitions are important.

Reply:
My main reasoning is cleaning up imports across SLSim. The previous implementation of LensPop had multiple imports from slsim.Deflectors and slsim.Sources, which can be cleaned up by treating these as submodules with relevant imports brought in via their respective __init__.py files.

I'm not sure I follow why this would be harder to maintain, unless you're referring to the re-exports and potential future name changes of classes within these submodules?

Reply:
Yes, I was meaning for further potential name changes or addition of classes, these init files would also need to be updated

Reply:
Currently we have multiple imports from within a single submodule. A rename of a class would require changes in the submodule as well as for any files that import the renamed class. If we instead make use of __init__.py, we have to make a similar amount of changes (with one additional change being in the submodule __init__.py), with the benefit of having a clear and readable import structure. A similar argument holds for addition of new classes or functions that we wish to be public facing.

I'm not sure I see the point in organizing things within submodules, if we do not make use of the __init__.py file while needing to access multiple files within said submodule.